modin/core/execution/ray/common/utils.py

```diff
@@ -82,7 +83,10 @@ def initialize_ray(
     # the `pandas` module has been fully imported inside of each process before
     # any execution begins:
     # https://github.com/modin-project/modin/pull/4603
-    env_vars = {"__MODIN_AUTOIMPORT_PANDAS__": "1"}
+    env_vars = {
```

Comment:
What if we connect to an already running Ray cluster? Will we see FutureWarnings?

Reply:
Unfortunately, yes, I don’t know of a way to change the state of an already running cluster on Ray.